android/libraries/rib-workflow-test/src/main/AndroidManifest.xml

```diff
@@ -1,2 +1,2 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest package="com.uber.rib.workflow.test"/>
+<manifest />
```

Comment:
Are safe to kill otherwise empty manifests?

Reply:
Let me check and get back on this

Reply:
I think this is recently allowed. [1](https://issuetracker.google.com/issues/242580918), [2](https://issuetracker.google.com/issues/232803637), [3](https://issuetracker.google.com/issues/232420210)

Will patch